### DISCUSSION TITLE
Is this still active?

### DISCUSSION THREAD

**Discussion by Steve0Greatness** [FlagClicked/FlagClicked#109] [Help]
I'm wondering if this repository is still actively maintained, or if people are still working on this in any way.

**Comment by danielgallab** [comment]
No, not really, but this is still something I'm interested in. If I end up ever doing this again, it will be on my account instead of in an org, since vercel would only be free if it was on a user repo. I was having trouble getting a MongoDB app running on vercel, but I'll try to explore it more and try to make this a reality again.
